src/main/java/org/jboss/maven/plugins/bombuilder/BuildBomMojo.java

```diff
@@ -69,7 +69,7 @@
      * BOM output file
      */
     @Parameter( defaultValue = "bom-pom.xml" )
-    private String outputFilename;
+    String outputFilename;
 
```

Comment:
I did not want to make the outputFilename publicly available or make it property. I just need to set the value in test, hence default visibility. If you prefer setter, I can change it